EnvironmentSetup/AWS/README.md

```diff
@@ -9,9 +9,9 @@ This document describes the setup of Amazon Kubernetes (EKS) and Wolfram Applica
 
 The following CLI tools are required to be installed on your local machine to complete the setup and installation:
```

Comment:
@delaix This Kubernetes and AKS version upgrade is going to be major release? Shall we move our version to 3.0.0?

Reply:
A major release means that things may not be compatible with prior releases. Are these changes that significant?

Reply:
Actually yes, ingress-controller 1.2.1 is not compatible with the previous cluster version. EFS images are different.

Reply:
OK, version 3 it is.